app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Model/User.java

```diff
@@ -85,6 +87,23 @@ public static void saveCredentials(Context context, String username, String pass
     }
 
     /**
+     * Save the user details of the logged in user in shared preferences
+     *
+     * @param context context of the activity that calls this method
+     * @param person model of logged in user
+     */
+    public static void saveUserDetails(Context context, Person person) {
+        if (person == null) {
```

Comment:
Good! `context` almost always shouldn't be null but since we can't trust our clients not to pass in a `null` `Context` we should probably check for that too!